filesystem/src/main/java/org/frankframework/filesystem/FileSystemListener.java

```diff
@@ -357,10 +357,10 @@ public Message extractMessage(@Nonnull RawMessageWrapper<F> rawMessage, @Nonnull
 			if (attributes!=null) {
 				messageProperties.putAll(attributes);
 			}
-			if (!"path".equals(getMessageType())) {
+			if (getMessageType() != MessageType.PATH) {
```

Comment:
How was this not caught by our tests? 🤔

Reply:
Since this was not caught by tests apparently, I actually wonder if the right fix should be 
```suggestion
			if (MessageType.PATH.name().equals(getMessageType().name())) {
```